documents/DeveloperGuide/GraphEditor.md

```diff
@@ -2,29 +2,30 @@
 
 The MaterialX Graph Editor is an example application for visualizing, creating, and editing MaterialX graphs.  It utilizes the ImGui framework as well as additional ImGui extensions such as the Node Editor.
 
-### Example Images
+## Example Images
+
+![MaterialX Graph Editor with procedural marble example](../Images/MaterialXGraphEditor_Marble.png)
```

Comment:
Not specific to this PR but I noticed that images don't show up properly on the `PyPi` package page for MaterialX.
If were adding more images, just curious is there a more robust way to specify image references -- such as an absolute reference to the web site location instead of using relative paths ?

Reply:
Oh, I see the issue: https://pypi.org/project/materialx/

It's great to see that the documentation content is reused in many places!

I've migrated the Markdown docs, including `README.md`, to use absolute paths now:

- https://github.com/AcademySoftwareFoundation/MaterialX/pull/1567/commits/e129d65ee2ba3d3acb9996c131da61c78731d165

Reply:
The update of these markdown files has been split off into its own PR: https://github.com/AcademySoftwareFoundation/MaterialX/pull/2037